recipes/nextstrain-cli/meta.yaml

```diff
@@ -40,9 +42,9 @@ requirements:
     # botocore. We can declare the boto3 dependency and s3fs here, since Conda
     # can resolve these dependency issues.
     # See: https://github.com/nextstrain/cli/blob/77c97bfecaf3dda84a8c20c343afc70cc6659f53/setup.py#L91-L98
-    - fsspec
+    - fsspec !=2023.9.1
```

Comment:
[Linter is raising the following](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=41603&view=logs&j=eb1ca712-fec5-5d93-deb0-40983fe61f2f&t=22a463be-d34e-5e0b-a210-c6720f3b9b60&l=28)
```
ERROR: recipes/nextstrain-cli/meta.yaml:45: version_constraints_missing_whitespace: Packages and their version constraints must be space separated
```
But this seems like a false positive to me.

Reply:
Hmm. Linting line numbers may be slightly off due to pre-processing (like the Jinja2 template processing)... I'd guess it's not complaining about

```yaml
    - fsspec !=2023.9.1
```

but maybe instead about:

```yaml
    - s3fs >=2021.04.0, !=2023.9.1
```

because Conda package specs are defined as up to three space-separated parts: name, version spec, build. The version spec part cannot contain spaces. See the [docs about Conda package specs](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications).

Does linting pass with

```diff
-    - s3fs >=2021.04.0, !=2023.9.1
+    - s3fs >=2021.04.0,!=2023.9.1
```

instead?

Reply:
This is a bug in the linter it seems, it doesn't accept constraints that start with `!`, so the line is correct, it's just a bug in the linter.

Fixed in https://github.com/bioconda/bioconda-utils/pull/919